src/RefractAPI.cc

```diff
@@ -342,14 +340,13 @@ namespace drafter {
         element->meta[SerializeKey::Title] = PrimitiveToRefract(MAKE_NODE_INFO(action, name));
 
         if (!action.node.relation.str.empty()) {
-            // FIXME: add SourceMap
-            element->attributes[SerializeKey::Relation] = refract::IElement::Create(action.node.relation.str);
+            refract::StringElement* relation = refract::IElement::Create(action.node.relation.str);
```

Comment:
Please add explaining comment why is used combo `Element::Create()` and `attachSourceMap()` 
instead of `PrimitiveToRefract`